### ISSUE TITLE
Leave invalid CSS after compilation option

### ISSUE THREAD

**Issue by mwojslaw** [sass/dart-sass#2244]
Hi, is it possible to leave invalid CSS after running sass compilation? Sass cli command is only part of our styling pipe, unfortunately one that have to be process at the beginning. We're making use of "deep selector" (vuejs term) which looks like this "& >>> li", and is processed by different software. When running sass cli this part of css is cut with message: "The selector "& > > > li" is invalid CSS. It will be omitted from the generated CSS.
This will be an error in Dart Sass 2.0.0". 

Can we somehow take this problem? I tried to use --silence-deprecation=bogus-combinators but without success.

**Comment by nex3** [comment]
No. This will be a syntax error in Dart Sass 2.0.0, which means that there won't be a way for us to even represent it internally. In fact, one of the major reasons we're deprecating this behavior is because the current internal representation of invalid combinators adds _major_ complications to Sass's selector logic.

I strongly recommend filing an issue against Vue to provide an alternative, CSS-compatible syntax for this. For example, Angular uses `::ng-deep` for [something similar](https://angular.io/guide/component-styles#component-styling-best-practices).

> I tried to use --silence-deprecation=bogus-combinators but without success.

This is a bug. I'll work on a fix.
